tests/actions.go

```diff
@@ -205,6 +206,7 @@ type OperatorActions interface {
 	CheckUpgradeCompleteOrDie(info *TidbClusterConfig)
 	CheckInitSQL(info *TidbClusterConfig) error
 	CheckInitSQLOrDie(info *TidbClusterConfig)
+	DeployAndCheckPump(tc *TidbClusterConfig) error
```

Comment:
where is its implementation?

Reply:
in `backup.go`, this is an existing function but was not included in the interface definition formerly.